PCP/server/src/api/cart_api.py

```diff
@@ -0,0 +1,42 @@
+from flask import jsonify
+from flask_restful import Resource
+
+try:
+    from src.db.cart import get_cart_contents
+except ImportError:
+    from db.cart import get_cart_contents, add_item_to_cart, remove_item_from_cart
+
+
+class Cart(Resource):
+    def get(self, user_id):
+        """Fetches all items in a user's shopping cart by their user ID."""
+        try:
+            cart_items = get_cart_contents(user_id)
+            if not cart_items:
+                return jsonify({"error": "No items found in the cart"}), 404
+            return jsonify({"cart_items": cart_items})
+        except Exception as e:
+            return jsonify({"error": str(e)}), 500
+
+
+class AddToCart(Resource):
```

Comment:
it should be get , post and put in one class u don't have to make two class over here please fix it